client/js/table-model.js

```diff
@@ -16,6 +16,14 @@ class TableModel {
 	setValue(location, value) {
 		this.data[this._getCellId(location)] = value;
 	}
+
+	addRow() {
+		this.numRows++;
```

Comment:
Sorry about that! I've converted the indentation of the existing files and have set my editor preferences to use two spaces.